### PR TITLE
Fix support overrides tags

### PR DESCRIPTION
**Pull request by ia3andy**
Closes redhat-support: 144 148 151 152

### CODE REVIEW

**Comment by ia3andy** [comment]
@maxandersen review?

File: bom/runtime/src/main/resources/extensions-support-overrides.json [hunk 1]
```diff
@@ -160,6 +184,14 @@
          "dev-support" ]
        }
     },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-logging-json",
```
Comment (rsvoboda) [inline bom/runtime/src/main/resources/extensions-support-overrides.json:189]: @tqvarnst mentioned that he might shift it to TP or unsupported level

Comment (ia3andy) [reply bom/runtime/src/main/resources/extensions-support-overrides.json:189]: @rsvoboda let me know, create a new issue in that case ;-)

**Comment by maxandersen** [comment]
syntax wise looks fine - its for @tqvarnst  and @rsvoboda to confirm the semantics.

File: bom/runtime/src/main/resources/extensions-support-overrides.json (hunk 1 above)
Comment (rsvoboda) [reply bom/runtime/src/main/resources/extensions-support-overrides.json:189]: quarkus-logging-json stays on supported level

**Comment by maxandersen** [comment]
@ia3andy can you confirm my rebase works and merge if it does ? :)

**Comment by ia3andy** [comment]
@maxandersen I am not allowed to merge

**Comment by maxandersen** [comment]
okey, can you check then i'll merge ;)

**Comment by ia3andy** [comment]
@maxandersen why is CI failing?

**Comment by maxandersen** [comment]
@ia3andy ci is failing due to quay.io being down. can you verify this PR works in code.quarkus as expected ? then i'll merge it.

**Comment by ia3andy** [comment]
@maxandersen I'll test it tomorrow, I am done for the day.. (maybe quay is going to be up then)

**Comment by ia3andy** [comment]
@maxandersen just tested it and it works fine!

**Comment by ia3andy** [comment]
[test]

**Pull request merged**